setup.py

```diff
@@ -15,7 +15,7 @@
 
 setup(
     name="gyp-next",
-    version="0.6.1",
+    version = "0.6.2",
```

Comment:
```suggestion
    version="0.6.2",
```

I checked locally and the spaces fail linting :(